src/store/actions.ts

```diff
@@ -0,0 +1,80 @@
+import { v4 } from 'uuid';
```

Comment:
as I know actions are plain objects like { type: '', payload: '' } and we are not using them in ReduxToolkit
so I would recommend moving the below logic to the relative components and removing this file